src/main/java/com/smartbear/jenkins/plugins/testcomplete/TcDynamicReportAction.java

```diff
@@ -150,8 +150,9 @@ public void doDynamic(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
 
                 inputStream = archive.getInputStream(targetEntry);
                 rsp.serveFile(req, inputStream, targetEntry.getTime(), 0, targetEntry.getSize(), targetEntry.getName());
-            }
-            catch (Exception e) {
+            } catch (RuntimeException e) {
```

Comment:
@osamasalem , The below handler should handle the runtime exception as well right. Is it intentional that we don't want to call sendError() for RuntimeException?

Reply:
Thanks @saivenkat999 ... Really appreciate you comment 🙏 

This is one of the spotbug comments under REC_CATCH_EXCEPTION
> This method uses a try-catch block that catches Exception objects, but Exception is not thrown within the try block, and RuntimeException is not explicitly caught. It is a common bug pattern to say try { ... } catch (Exception e) { something } as a shorthand for catching a number of types of exception each of whose catch blocks is identical, but this construct also accidentally catches RuntimeException as well, masking potential bugs.
> A better approach is to either explicitly catch the specific exceptions that are thrown, or to explicitly catch RuntimeException exception, rethrow it, and then catch all non-Runtime Exceptions, as shown below:
``` 
try {
    ...
} catch (RuntimeException e) {
    throw e;
} catch (Exception e) {
    ... deal with all non-runtime exceptions ...
}
```

I have something in my gut with this solution ... but I am wondering if there is any better ?